pkg/products/datasync/reconciler.go

```diff
@@ -78,6 +79,8 @@ func NewReconciler(configManager config.ConfigReadWriter, installation *integrea
 
 	logger := logrus.NewEntry(logrus.StandardLogger())
 	var httpClient http.Client
+	httpClient.Timeout = time.Second * 10
+	httpClient.Transport = &http.Transport{DisableKeepAlives: true, IdleConnTimeout: time.Second * 10}
```

Comment:
Do we need `IdleConnTimeout` if keep alive connections are disabled? If I understand correctly, that timeout is only used to close idle connections, according to the doc:

```
    // IdleConnTimeout is the maximum amount of time an idle
    // (keep-alive) connection will remain idle before closing
    // itself.
    // Zero means no limit.
```